components/tag-list/tag-list-item-mixin.js

```diff
@@ -0,0 +1,86 @@
+import '../colors/colors.js';
+import { css, html } from 'lit-element/lit-element.js';
+import { labelStyles } from '../typography/styles.js';
+
+export const TagListItemMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * @ignore
+			 */
+			role: { type: String, reflect: true },
+			/**
+			 * @ignore
+			 */
+			tabindex: { type: Number, reflect: true }
```

Comment:
I find if I remove this, this doesn't get set as an attribute on the custom element and then I can't set the focus through JS.  But maybe I'm missing something?

Reply:
Hmm, maybe a casing thing. The JS property is `tabIndex` -- I'd set that in the constructor and later when you want to change it instead of the attribute.

Reply:
Yeah menu-items confused me because they both sprout it, and menu adds it.  Looks like it was a casing issue, but I could also remove it from the items altogether and have the list add it to every single one.

Reply:
Oh wow... `MenuItemMixin` also uses lower case.  We should probably correct that.  Probably isn't an issue since `d2l-menu` sets it anyway.  Looks like `d2l-tab-internal` uses the correct case.

Reply:
So it looks like our constructor tests hate this, which is maybe why `MenuItemMixin` uses lowercase?  And `d2l-tab-internal` [doesn't have a constructor test](https://github.com/BrightspaceUI/core/blob/main/components/tabs/test/tabs.test.js#L18)...

Reply:
So it would probably 💥 if it were constructed with `createElement`...
![image](https://user-images.githubusercontent.com/9042472/159083714-5388546d-43be-43d4-8ffa-38d0d1b153e7.png)
`MenuItemMixin` using lower case seems like a mistake, since it wouldn't really do anything.  We just never noticed because `d2l-menu` assigns the attribute anyway.